ratbagd/ratbagd-device.c

```diff
@@ -195,13 +218,13 @@ static int ratbagd_device_commit(sd_bus_message *m,
 }
 
 static int
-ratbag_device_get_model(sd_bus *bus,
-			   const char *path,
-			   const char *interface,
-			   const char *property,
-			   sd_bus_message *reply,
-			   void *userdata,
-			   sd_bus_error *error)
+ratbagd_device_get_model(sd_bus *bus,
```

Comment:
What is the difference between the model and the name?

Reply:
don't know, I just fixed the name. it used to be without the d at the end, which clashed with another function iirc.